src/lektorium/client/src/components/ControlPanel.vue

```diff
@@ -270,12 +281,12 @@ export default {
       this.edit_sessions = result.data.data.editSessions;
       this.parked_sessions = result.data.data.parkedSessions;
       this.checkStarting();
-      this.isLoading = false;
+      this.finishLoadingModal(timer_id);
     },
 
     async destroySession(session) {
       let id = session.sessionId;
-      this.isLoading = true;
+      let timer_id = this.startLoadingModal();
```

Comment:
I see lots of copy/pastte in form start/request/finish, could you please refactor it to separate function?